### PR TITLE
Spelling and Grammar Fixes and Tweaks

### PR DESCRIPTION
**Pull request by klorpa**
## About The Pull Request
Fixes various spelling and grammar errors, and made other spellings consistent.
## Why It's Good For The Game
Improves readability.
## Changelog
:cl:
spellcheck: fixed a few typos
/:cl:

### CODE REVIEW

File: code/game/atoms.dm [hunk 1]
```diff
@@ -600,7 +600,7 @@
 /**
  * React to an EMP of the given severity
  *
- * Default behaviour is to send the [COMSIG_ATOM_EMP_ACT] signal
+ * Default behavior is to send the [COMSIG_ATOM_EMP_ACT] signal
```
Comment (JohnFulpWillard) [inline code/game/atoms.dm:603]: This is not a misspelling and goes against our [style guideline](https://github.com/tgstation/tgstation/blob/2f1ddbef0194c0a241f2f975a016effcae139f47/.github/guides/STYLE.md#things-that-do-not-matter)

File: code/game/atoms.dm [hunk 2]
```diff
@@ -1083,16 +1083,16 @@
...
 /atom/proc/on_log(login)
 	if(loc)
 		loc.on_log(login)
 
 
 /*
-	Atom Colour Priority System
-	A System that gives finer control over which atom colour to colour the atom with.
+	Atom Color Priority System
+	A System that gives finer control over which atom color to color the atom with.
```
Comment (JohnFulpWillard) [inline code/game/atoms.dm:1095]: ^

**Comment by SuperNovaa41** [comment]
This is a pointless pr

File: code/game/atoms.dm (hunk 1 above)
Comment (SuperNovaa41) [reply code/game/atoms.dm:603]: This whole PR is just this aside from a handful of actual in game text. i think it should be closed

**Comment by san7890** [comment]
This PR has been conflicted for 10 days, and stale for almost half a week. Since it goes against our style guide (found at https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#things-that-do-not-matter), it will be closed to keep the PR tracker clean. Feel free to re-PR this change with grammar fixes+changes that could matter in the future.

**Pull request closed without merging**